Pipfile

```diff
@@ -14,6 +14,7 @@ mypy = "==0.770"
 
 [packages]
 flask = "==1.1.2"
+itsdangerous = "==1.1.0"
```

Comment:
Maybe just add a comment saying that it is a dependency of flask and should be remove with the next flask version